docs/UserGuide.md

```diff
@@ -43,144 +43,351 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
+Format: `find [TYPE] KEYWORD [MORE_KEYWORDS]`
+
+> * Type is optional.
+> * If type is not specified, 3 separated lists of matched events, deadlines and tasks will appear.
+> * Type should be `ev`, `dl` or `ts`.
+> * The search is case insensitive. e.g `discussion` will match `Discussion`
+> * The order of the keywords does not matter. e.g. `Project Discussion` will match `Discussion Project`
+> * Only the description is searched.
+> * Events/deadlines/tasks matching at least one keyword will be returned (i.e. `OR` search).
+    e.g. `Project` will match `Project Discussion`
 
 Examples:
 
-* `edit 1 p/91234567 e/johndoe@yahoo.com`<br>
-  Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@yahoo.com` respectively.
+* `find project discussion`<br>
+  Returns 3 separated lists of events/deadlines/tasks having description `project` or `discussion`.
 
-* `edit 2 Betsy Crower t/`<br>
-  Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `find ts survey`<br>
+  Returns a task list of tasks having description `survey`.
 
-### 2.5. Finding all persons containing any keyword in their name: `find`
+### 2.6. Finding all events/deadlines happening on a certain day : `on`
 
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Finds events and deadlines which happen on the given date.<br>
+Format: `on DATE`
 
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+> * DATE should have format `DD-MM-YY`, `DD-MM` or `DD`.
+> * If `DD-MM-YY` is entered, 2 separated lists of events and deadlines on the entered date will appear.
+> * If `DD-MM` is entered, 2 separated lists of events and deadlines on the entered date in any year will appear.
+> * If `DD` is entered, 2 separated lists of events and deadlines on the entered date in any month and any year will appear.
 
 Examples:
 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `on 23-04`<br>
+  Returns 2 separated lists of all events and deadlines happening on 23rd April in any year.
 
-### 2.6. Deleting a person : `delete`
+### 2.7. Viewing an event/deadline/task : `view`
 
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
+Views the specified event/deadline/task form the event/deadline/task list.
 
-> Deletes the person at the specified `INDEX`. <br>
-> The index refers to the index number shown in the most recent listing.<br>
-> The index **must be a positive integer** 1, 2, 3, ...
+Format: `view TYPE INDEX`
```

Comment:
Like in `edit`, tie the `INDEX` for `delete` with the current tab, since we can assume that the user would need to `list` the correct task type.  So can remove the `TYPE`